content/en/synthetics/ci_results_explorer.md

```diff
@@ -28,55 +30,63 @@ You can:
 * Keep track of the progress of your tests in CI pipelines.
 * Identify flaky tests to fix.
 
-{{< img src="synthetics/ci_results_explorer/ci_results_explorer.png" alt="CI Results Explorer" style="width:100%;">}}
-
-## Exploring batches
+## Explore test batches
 
-The CI Results Explorer displays batches of tests that were run through [Synthetics and CI/CD][1]. Every batch corresponds to a call to the Datadog API (through the NPM package or directly through the API endpoint) to trigger one or several test executions. 
+The CI Results Explorer displays batches of tests run by the [Synthetics and CI/CD integration][1]. Each batch corresponds to a call to the Datadog API (through the NPM package or directly through the API endpoint) and triggers one or several test executions. 
```

Comment:
```suggestion
The CI Results Explorer displays batches of tests run by [Synthetics and your CI/CD provider][1]. Each batch corresponds to a call to the Datadog API (through one of our [CI/CD integrations][1], the [datadog-ci][3] NPM package or directly through the API endpoint) and triggers one or several test executions. 
```

Let me know what you think

Reply:
Also would need to add the link after line 96: https://www.npmjs.com/package/@datadog/datadog-ci